1.segment_images/scripts/2.segment_whole_organoids.py

```diff
@@ -70,8 +70,10 @@
     input_dir = pathlib.Path(args.input_dir).resolve(strict=True)
 
 else:
-    input_dir = pathlib.Path("../../data/z-stack_images/C4-2/").resolve(strict=True)
-    window_size = 3
+    input_dir = pathlib.Path("../../data/NF0014/zstack_images/C4-2/").resolve(
+        strict=True
+    )
+    window_size = 5
```

Comment:
It is interesting that this sliding window matches the nuclei sliding window. Do you find this is performing better?